app/controllers/application_controller.rb

```diff
@@ -85,7 +85,7 @@ def prompt_to_enter_otp
   end
 
   def track_get_requests
-    return unless request.get?
+    return unless request.get? && request.fullpath != '/active'
```

Comment:
this smells a little like breaking encapsulation. I.e. should this method need to know that much detail about specific requests?

does `skip_after_action` work http://api.rubyonrails.org/classes/AbstractController/Callbacks/ClassMethods.html#method-i-skip_after_action ?